src/sql/workbench/services/queryEditor/browser/queryEditorService.ts

```diff
@@ -87,8 +87,9 @@ export class QueryEditorService implements IQueryEditorService {
 		let docUri: URI = URI.from({ scheme: Schemas.untitled, path: filePath });
 
 		// Create a sql document pane with accoutrements
-		const fileInput = this._untitledEditorService.create({ associatedResource: docUri, mode: 'sql' });
+		const fileInput = this._untitledEditorService.create({ untitledResource: docUri, mode: 'sql' });
 		const m = await fileInput.resolve();
+		m.unregisterWorkingCopy();
```

Comment:
do you know why this is now necessary and wasn't before?

Reply:
From what I'm seeing, when I close the editor, the function call at the end "toDisposable" does not fire when I close ADS but it does when closing the editor manually, otherwise it seems to automatically backup the editor into the backup file service for similar reasons as notebooks in #8944